Directory.Build.props

```diff
@@ -14,6 +14,9 @@
     <AnalysisMode>Recommended</AnalysisMode>
     <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
     <DotNetUseShippingVersions>true</DotNetUseShippingVersions>
+    <RestoreUseStaticGraphEvaluation>true</RestoreUseStaticGraphEvaluation>
```

Comment:
It's not clear to me why this -TFM suffix feature needs nuget static graph restore. Doesn't it work without? If not then this is probably a bug in Microsoft.DotNet.Build.Tasks.TargetFramework.nupkg.

Reply:
No, it does not work without this: The targets I brought from runtime were not getting reached. Then I noticed the NuGet path I mentioned in chat was how those targets were getting reached in runtime, and now they're working here.

Reply:
Please investigate why it doesn't work without static graph restore. This might be a bug that we should fix.

Reply:
Ok I removed it and now it works. What!